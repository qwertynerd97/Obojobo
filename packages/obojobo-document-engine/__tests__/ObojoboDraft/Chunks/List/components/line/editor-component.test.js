import React from 'react'
import renderer from 'react-test-renderer'

import Line from 'ObojoboDraft/Chunks/List/components/line/editor-component'

describe('Line Editor Node', () => {
	test('Line component', () => {
		const component = renderer.create(
			<Line
				node={{
					data: {
						get: () => {
							return {}
						}
					}
				}}
			/>
		)
		const tree = component.toJSON()

		expect(tree).toMatchSnapshot()
	})
})
