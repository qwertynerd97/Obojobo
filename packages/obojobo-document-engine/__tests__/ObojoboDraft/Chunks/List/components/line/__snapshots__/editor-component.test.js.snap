// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Line Editor Node Line component 1`] = `
<div>
  <li />
</div>
`;
